girder/utility/__init__.py

```diff
@@ -34,6 +37,30 @@ def camelcase(value):
                    re.split("[._]+", value))
 
 
+def mkdir(path, mode=0o777, recurse=True, safe=True):
+    """
+    Create a new directory or ensure a directory already exists.
+
+    :param path: The directory to create.
+    :type path: str
+    :param mode: The mode (permission mask) prior to applying umask.
+    :type mode: int
+    :param recurse: Whether intermediate missing dirs should be created.
+    :type recurse: bool
+    :param safe: Set to True to suppress the error if the dir already exists.
```

Comment:
@zachmullen IMO, ``safe`` isn't a really clear name for a functionality which suppresses errors (I'd expect a safe mode to do more error checking).

[``os.makedirs`` in Python 3.2+ has the ``exist_ok`` parameter](https://docs.python.org/3/library/os.html#os.makedirs), which does the exact same thing as this parameter. How about just change the name to something like that?